react/features/base/config/reducer.js

```diff
@@ -189,11 +187,6 @@ function _setConfig(state, { config }) {
 function _translateLegacyConfig(oldValue: Object) {
     const newValue = oldValue;
 
-    if (!Array.isArray(oldValue.toolbarButtons)
```

Comment:
I think removing this will make the backwards compatible code only work from URL params and not from interface_config.js